.devcontainer/Dockerfile

```diff
@@ -43,7 +43,9 @@ RUN apt-get -y update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
     python3 \
     python3-pip \
     unzip \
-    wget
+    wget \
```

Comment:
I see wget now being used in the diff, but what are `file` & `desktop-file-utils` for ?